pkg/consts/consts.go

```diff
@@ -44,6 +44,7 @@ const (
 	MethodNameFeeQuoterGetTokenPrice        = "GetTokenPrice"
 	MethodNameGetFeePriceUpdate             = "GetDestinationChainGasPrice"
 	MethodNameFeeQuoterGetStaticConfig      = "GetStaticConfig"
+	MethodNameFeeQuoterGetDynamicConfig     = "GetDynamicConfig"
```

Comment:
afaict `GetDynamicConfig` is not a method of FeeQuoter, I do see it in OffRamp and OnRamp though.